parsers/convert-font/convert-font.parser.ts

```diff
@@ -46,11 +47,17 @@ export default async function (
 
     return (
       await Promise.all(
-        designTokens.map(async designToken => {
-          const signedUrlsByFormat = await SpServices.font.convert({
-            postscriptName: designToken.value.fontPostScriptName,
-            formats,
-          });
+        designTokens.flatMap(async designToken => {
+          const signedUrlsByFormat = await new Promise<PartialRecord<AllowedFormat, string>>(
```

Comment:
Why not use a try catch ?

Reply:
I keep `try catch` for global catch and I don't want to use nested try catch.